src/app/transaction-spending-limit/transaction-spending-limit.component.html

```diff
@@ -43,6 +43,15 @@
     *ngIf="transactionSpendingLimitResponse.DAOCoinLimitOrderLimitMap"
   >
   </app-transaction-spending-limit-section>
+  <div *ngif="transactionSpendingLimitResponse.IsUnlimited">
+    <div class="d-flex flex-column">
+      <div class="d-flex">
+        <div class="d-flex justify-content-start align-items-center col-5">
+          This key has FULL ACCESS.
```

Comment:
We should probably explain this a little more here.

Reply:
also we should not show the limit sections (cc, DAO coin, NFT, etc.) if the key is unlimited.